src/config.js

```diff
@@ -0,0 +1,16 @@
+
+const port = parseInt(process.env.PORT, 10) || 3030;
+const appSid = process.env.APP_SID;
+const accountSid = process.env.ACCOUNT_SID;
+const apiKey = process.env.API_KEY;
```

Comment:
Nitpick, these are technically API Key Pairs, which consist of an API Key SID and API Key Secret. Should we use `API_KEY_SID` and `API_KEY_SECRET`?

Reply:
Updated